test/unit/resources/docker_container_test.rb

```diff
@@ -24,6 +24,7 @@
       _(resource.tag).must_equal '12.04'
       _(resource.command).must_equal '/bin/bash'
       _(resource.ports).must_equal ''
+      _(resource.labels).must_equal ['app=example', 'version=1.5.4']
```

Comment:
This currently has a failing unit test, which needs to be fixed.

Also, you mentioned that there was an existing unit test which tested a container with zero labels; could you show me where that is?

Reply:
Yup, looks like I needed to fix the `docker_container` resource since `labels[0]` is no longer correct.

Also, I didn't see where a test where a docker container was explicitly returning `[]` when no labels are used. Couldn't hurt to have an explicit test instead of relying on `docker.containers.labels` being correct. I'll make it so.